src/main/java/com/debatetimer/domain/organization/Organization.java

```diff
@@ -0,0 +1,26 @@
+package com.debatetimer.domain.organization;
+
+import java.util.List;
+import lombok.Getter;
+
+@Getter
+public class Organization {
```

Comment:
혹시, Lombok 안쓴 이유가 따로 있나요?

```suggestion
@RequiresArgsConstructor
public class Organization {
```

Reply:
`@RequiresArgsConstructor`는 Controller, Service, DomainRepository와 같은 계층형 객체에만 사용하기로 했었습니다. 도메인 객체에는 생성자에 String, int와 같은 전형적인 타입이 들어오기 때문에 일부러 사용하지 않았습니다. (그렇게 합의하기로 했던 것 같습니다.)
<img width="751" height="330" alt="image" src="https://github.com/user-attachments/assets/724db4e6-9371-412e-90e6-7ec0ea3ddb23" />